sdks/java/core/src/test/java/org/apache/beam/sdk/util/PubsubJsonClientTest.java

```diff
@@ -115,7 +118,7 @@ public void publishOneMessage() throws IOException {
         .encodeData(DATA.getBytes())
         .setAttributes(
             ImmutableMap.of(TIMESTAMP_LABEL, String.valueOf(MESSAGE_TIME),
```

Comment:
This is increasingly difficult to read as you add more to it. Consider swapping to `ImmtuableMap.builder().put(...)...put(...).build()`